tests/e2e/tests/memcache_test.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for the memcache API.
+
+This test module is incomplete but is a starter.
+"""
+
+import pytest
+import time
```

Comment:
standard library imports go above third-party imports in a separate section. 

(I need to setup linting for tests)